tests/routes/test_documents_deprecated.py

```diff
@@ -0,0 +1,633 @@
+from app.db.models.deprecated import (
```

Comment:
How come this is a new module rather than a rename?

Have the existing tests changed that much for the deprecated documents?

Reply:
This file was just a rename from the old `test_documents.py`